src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerCoordinator.swift

```diff
@@ -646,11 +646,8 @@ class QRScannerCoordinator {
 		}
 
 		self.recycleBin.restore(recycleBinItem)
-
-		self.parentViewController?.dismiss(animated: true) {
-			let exposureSubmissionCoordinator = self.exposureSubmissionCoordinator(parentViewController: parentViewController)
-			exposureSubmissionCoordinator.start(with: coronaTest.type)
-		}
+		let exposureSubmissionCoordinator = self.exposureSubmissionCoordinator(parentViewController: parentViewController)
```

Comment:
Is this also working for the other cases like .submissionFlow, .onBehalfFlow ?

Reply:
I checked it with the submission flow but i will check again